test/functional/bundlelist/all/list/test.bats

```diff
@@ -13,10 +13,8 @@ teardown() {
   clean_tars 10
 }
 
-@test "bundle-add list bundles" {
-  run sudo sh -c "$SWUPD bundle-add $SWUPD_OPTS --list"
-
-  check_lines "$output"
+@test "bundle-list all bundles" {
+  run sudo sh -c "$SWUPD bundle-add --all"
```

Comment:
Can you restore the `check_lines` call? It's useful to verify swupd's expected printed output to actual output.